contracts/implementations/Mapper.sol

```diff
@@ -18,4 +53,21 @@ contract Mapper {
             sourceNode
         ];
     }
+
+    /// @notice Gets the beneficiary associated with the pair idProxy/nodeId
```

Comment:
Maybe a note like “if the beneficiary is not explicitly set, it defaults to the owner”.